crates/rooch/src/commands/transaction/commands/build.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::cli_types::{CommandAction, TransactionOptions, WalletContextOptions};
+use async_trait::async_trait;
+use move_core_types::{
+    account_address::AccountAddress,
+    identifier::Identifier,
+    language_storage::{ModuleId, TypeTag},
+};
+use moveos_types::{move_types::FunctionId, transaction::MoveAction};
+use rooch_types::error::RoochResult;
+
+/// Get transactions by order
+#[derive(Debug, clap::Parser)]
+pub struct BuildCommand {
```

Comment:
Add a `output` option to allow write the tx to a file, for the `sign` command.

Reply:
Updated `output` as `bool`.